### PR TITLE
adds 2nextprez content

### LINKED ISSUE #1682: Create initiative landing page for Letters to the Next President
We'll need a landing page that we can point people to based on our content, events, etc. related to our involvement in NWP's [Letters to the Next President](http://letters2president.org/) campaign. 

**Deadline**
* End of April (so we have it ready for May Curriculum Workshop)

**Content**
* Partner logo - find campaign logos here https://www.dropbox.com/sh/1xat5jzm23eqak2/AABBmue-PFmLbiz2KnKwa2q5a?dl=0
* Title
* Intro text
* Thumbnails and links to 3+ activities
* List and links to up to 3 upcoming/related events (this section would eventually be removed - after the election)
* Gallery of remixes (this would be a "nice to have")

I know this may still be a consideration for a CMS template - let me know if you need further details.

cc @chadsansing

### PR DESCRIPTION
**Pull request by hannahkane**
Fixes #1682

**Changes proposed in this pull request:**
- adds new activity module page, including images
- updates Activities landing page to include section on the new Letters to the Next President content at the top
- creates a route for the new page

### CODE REVIEW

**Comment by hannahkane** [comment]
@Pomax - would you have time to review this, by any chance?

File: pages/next-prez.jsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        caption: "screencap by Mozilla from a video by Hypothes.is",
+        subtitle: "Compose, Evaluate, Remix",
+        description: "Use the web-native annotation tool Hypothes.is to share insights and analysis as you read and annotate peers’ Letters to the Next President projects.",
+        link: "https://thimbleprojects.org/chadsansing/30045/#overview"
+      },
+    ]
+  },
+];
+
+var curriculum = curriculumList.map(function (section) {
```
Comment (Pomax) [inline pages/next-prez.jsx]: nice ES6 syntax we can now use:
```
var curriculum = curriculumList.map(section => <ActivitySection title={section.title} key={section.title} activities={section.activities} />);
```

**Comment by Pomax** [comment]
This looks good to me, the ES6 change isn't really necessary, but does make things a little more intuitive to read. Your call! \o/

**Comment by hannahkane** [comment]
I'm going to merge this as is, but will make a note to learn more about ES6 soon! :)

**Pull request merged**